app/pb_kits/playbook/pb_form/form_builder/checkbox_field.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbForm
+    module FormBuilder
+      module CheckboxField
+        def check_box(name, data: {}, props: {})
+          value = props[:value] ||= 1
+          input = super(name, { required: props[:required], data: data }, value)
```

Comment:
One of things we worked hard with the other inputs has been to make the form system as thin of a wrapper over what rails provides out of the box as possible, and that means forwarding all none props to the super class. I would suggest you capture them like this.

```suggestion
        def check_box(name, props: {}, **options, &block)
          value = props[:value] ||= 1
          input = super(name, **options, &block)
```

Reply:
Ahh okay, I'll make those changes.  That super method was a bit of a black box in terms of which arguments it was parsing and in what order.  I'll mostly ended up with line 9 by trial and error.  I'll make those changes.  Should clean things up and help make it more uniform/conventional.

Reply:
@terryfinn updated to use the double asterisk options argument.  Everything looks good except the value attribute isn't applied to the html input even after including `options[:value] = props[:value]`.  Where exactly is the `super` method defined?  Would make it easier to debug if I could see how the arguments are being parsed.

Reply:
Quick follow up.  Looks like it's the default behavior to override any value attribute on a checkbox input and replace it with `1`.  So things are working as expected.  Devs will just have to work with a `1` - `0` boolean value system.